pkg/sql/ttl/ttljob/ttljob.go

```diff
@@ -247,17 +244,33 @@ func (t rowLevelTTLResumer) Resume(ctx context.Context, execCtx interface{}) err
 	}
 
 	if ttlSettings.RowStatsPollInterval != 0 {
+
 		g.GoCtx(func(ctx context.Context) error {
+
+			handleError := func(err error) error {
```

Comment:
Moved the error -> warning conversion up a layer to remove a layer of nesting inside `fetchStatistics`.